crates/bevy_color/src/lib.rs

```diff
@@ -149,5 +152,19 @@ where
     Self: From<Oklcha> + Into<Oklcha>,
     Self: From<Xyza> + Into<Xyza>,
     Self: Alpha,
+    Self: Add<Self>,
+    Self: Sub<Self>,
+    Self: Div<f32>,
+    Self: Mul<f32>,
+    f32: Mul<Self>,
+    Self: Point,
 {
```

Comment:
```suggestion
    Self: AddAssign<Self>,
    Self: SubAssign<Self>,
    Self: DivAssign<f32>,
    Self: MulAssign<f32>,
    f32: MulAssign<Self>,
    Self: Neg,
{
```

If we're going to do this, we might as implement all the relevant traits. `Neg` may look scary, but since we're already allowing scaling by an `f32`, negation is already possible via `-1. * color`, so we might as well provide the more intuitive version of that operation.